.github/workflows/_deploy_kafka_test.yml

```diff
@@ -0,0 +1,77 @@
+name: Deploy Kafka Test
+
+on:
+  workflow_call:
+    inputs:
+      cluster-name:
+        required: true
+        description: AKS Cluster Name
+        type: string
+      key-release-image: 
```

Comment:
I would give this a default

Reply:
I don't think we should give defaults image tags. Because images always evolves and 7 months later, the default might not even be close to the most updated one. I don't think we should come back to update the default image tag again. The entire purpose of the pipeline is to test an image tag and whether that image works. So whoever run the pipeline should have full control over what images are used. 

Another suggestion is to make the default image tags as an env var and have the default reference the env var so we can just update the env var to avoid updating the code when most updated working image tags change.

Reply:
Honestly since this is just called by kafka_demo_test, the default there will pass to these, so a default here isn't necessary, I just thought it might make it easier to see what the default is without having to go back to the other file. The purpose of the default is for when we're not running these manually, like in a PR.